### PR TITLE
[Component with its own class name as service type ] LPS-193506 Refactor Provider components from Message-boards

### PR DESCRIPTION
**Pull request by GabrielHenriquedev**

### CODE REVIEW

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

File: modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/util/MBMessageFormatUploadHandlerProviderRegistryUtil.java [hunk 1]
```diff
@@ -3,38 +3,36 @@
...
-import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.framework.FrameworkUtil;
 
 /**
  * @author Alejandro Tardín
  */
-@Component(service = MBMessageFormatUploadHandlerProvider.class)
-public class MBMessageFormatUploadHandlerProvider {
+public class MBMessageFormatUploadHandlerProviderRegistryUtil {
```
Comment (rafaprax) [inline modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/util/MBMessageFormatUploadHandlerProviderRegistryUtil.java:19]: Hey @GabrielHenriquedev, rename it to `MBMessageFormatUploadHandlerRegistryUtil`.

File: modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/util/MBMessageFormatUploadHandlerProviderRegistryUtil.java [hunk 2]
```diff
@@ -3,38 +3,36 @@
...
 
-	public MBMessageFormatUploadHandler provide(String format) {
+	public static MBMessageFormatUploadHandler provide(String format) {
```
Comment (rafaprax) [inline modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/util/MBMessageFormatUploadHandlerProviderRegistryUtil.java:21]: Hey @GabrielHenriquedev, rename the method to `get`. Thx

**Comment by GabrielHenriquedev** [comment]
ci:test:sf

**Comment by GabrielHenriquedev** [comment]
ci: test:revelevant

**Comment by GabrielHenriquedev** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:lima

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by Kvale1733** [comment]
ci:test:message-boards

**Comment by catarineaguiar** [comment]
no regressions found

File: modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/upload/format/MBMessageFormatUploadHandlerRegistryUtil.java [hunk 3]
```diff
@@ -8,33 +8,30 @@
...
+public class MBMessageFormatUploadHandlerRegistryUtil {
 
-	public MBMessageFormatUploadHandler provide(String format) {
+	public static MBMessageFormatUploadHandler get(String format) {
 		return _serviceTrackerMap.getService(format);
 	}
 
-	@Activate
-	protected void activate(BundleContext bundleContext) {
+	private static final ServiceTrackerMap<String, MBMessageFormatUploadHandler>
```
Comment (rafaprax) [inline modules/apps/message-boards/message-boards-web/src/main/java/com/liferay/message/boards/web/internal/upload/format/MBMessageFormatUploadHandlerRegistryUtil.java:24]: Hey @GabrielHenriquedev, please inline this serviceTracker into its unique usage. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Pull request closed without merging**